### ISSUE TITLE
Database extension requirements

### ISSUE THREAD

**Issue by peter279k** [izniburak/pdox#70]
As title, it seems that this package requires `PDO` extension.

And the `PDO` should have PDO database extensions.

For example, the MySQL is with `PDO` support and it should require `pdo_mysql`.

@izniburak, should we add some suggest block on `composer.json`?

**Comment by izniburak** [comment]
@peter279k I think this can be unnecessary. Because, someone who used that library may be using one of MySQL, PostgreSQL or SQLite databases. All extension should not be required for those databases. I had already added `ext-pdo` extension requirement in composer.json file. I think this is enough.
You think?

**Comment by peter279k** [comment]
@izniburak, thanks for your reply.

I mean we can consider letting these possible extensions on [suggest block](https://getcomposer.org/doc/04-schema.md#suggest).

And this can remind developers required extensions if they want to use one of database with PDO supports.

**Comment by izniburak** [comment]
Yes! This is make sense. We can add required extensions to suggest block.
